spec/unit/shoulda/matchers/active_record/have_attached_matcher_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'unit_spec_helper'
+
+# rubocop:disable Metrics/BlockLength
+describe Shoulda::Matchers::ActiveRecord::HaveAttachedMatcher,
+  type: :model do
+
+  if rails_gte_5_2?
+    before do
+      create_table :active_storage_blobs do |t|
+        t.string   :key,        null: false
+        t.string   :filename,   null: false
+        t.string   :content_type
+        t.text     :metadata
+        t.bigint   :byte_size,  null: false
+        t.string   :checksum,   null: false
+        t.datetime :created_at, null: false
+
+        t.index [ :key ], unique: true
+      end
+
+      create_table :active_storage_attachments do |t|
+        t.string     :name,     null: false
+        t.references :record,   null: false, polymorphic: true, index: false
+        t.references :blob,     null: false
+
+        t.datetime :created_at, null: false
+
+        t.index [ :record_type, :record_id, :name, :blob_id ], name: "index_active_storage_attachments_uniqueness", unique: true
+        t.foreign_key :active_storage_blobs, column: :blob_id
+      end
+
+      create_table(:users) do |t|
```

Comment:
When you use `define_model`, it will create a table for the model for you, so you don't usually need to do this.

Reply:
Done